modules/web-console/frontend/app/primitives/form-field/index.scss

```diff
@@ -276,6 +267,10 @@
                 right: 36px;
             }
         }
+        & > select.ng-invalid.ng-touched:not([multiple]) {
+            background-position: calc(100% - 36px) 50%;
+            padding-right: calc(28px + 26px);
```

Comment:
@Klaster1 Why `calc` in `padding-right` if we know the exact pixels number? Won't just `padding-right: 54px` do the thing?

Reply:
28px and 26px are used elsewhere too, I wanted to convey that, but was too lazy to extract the all the values into variables.

Reply:
Ok. I see.